product_docs/docs/epas/15/fundamentals/sql_fundamentals/02_conventions.mdx

```diff
@@ -7,13 +7,15 @@ redirects:
 
 <div id="conventions" class="registered_link"></div>
 
-The conventions used in the documentation apply to both Linux and Windows systems. Directory paths are presented in the Linux format with forward slashes. When working on Windows systems, start the directory path with the drive letter followed by a colon, and substitute back slashes for forward slashes.
+The conventions used in the documentation described here apply to both Linux and Windows systems. 
 
-Some of this information might apply interchangeably to the PostgreSQL and EDB Postgres Advanced Server database systems. The term *EDB Postgres Advanced Server* is used to refer to EDB Postgres Advanced Server. The term *Postgres* is used to generically refer to both PostgreSQL and EDB Postgres Advanced Server. When a distinction needs to be made between these two database systems, the specific names&mdash;PostgreSQL or EDB Postgres Advanced Server&mdash;are used.
+Directory paths are presented in the Linux format with forward slashes. When working on Windows systems, start the directory path with the drive letter followed by a colon, and substitute back slashes for forward slashes.
 
-The installation directory path of the PostgreSQL or EDB Postgres Advanced Server products is referred to as *POSTGRES_INSTALL_HOME*.
+Some of this information might apply interchangeably to the PostgreSQL and EDB Postgres Advanced Server (EPAS) database systems. The term *EDB Postgres Advanced Server* is used to refer to EDB Postgres Advanced Server. The term *Postgres* is used to generically refer to both PostgreSQL and EPAS. When a distinction needs to be made between these two database systems, the specific names&mdash;PostgreSQL or EDB Postgres Advanced Server&mdash;are used.
+
+The installation directory path of the PostgreSQL or EPAS products is referred to as *POSTGRES_INSTALL_HOME*.
 
 -   For PostgreSQL Linux installations, the default is `/opt/PostgreSQL/x.x` for version 10 and earlier. For later versions, use the PostgreSQL community packages.
--   For EDB Postgres Advanced Server Linux installations performed using the interactive installer for version 10 and earlier, the default is `/opt/edb/asx.x`.
--   For EDB Postgres Advanced Server Linux installations performed using an RPM package, the default is `/usr/edb/asxx`.
--   For EDB Postgres Advanced Server Windows installations, the default is `C:\Program Files\edb\asxx`. The product version number is represented by `x.x` or by `xx` for version 10 and later.
+-   For EPAS Linux installations performed using the interactive installer for version 10 and earlier, the default is `/opt/edb/asx.x`.
```

Comment:
I think we can remove this bullet. EPAS 10 is EOL'ed as are the interactive installers for Linux.
```suggestion
-   For EPAS Linux installations performed using the interactive installer for version 10 and earlier, the default is `/opt/edb/asx.x`.
```